web/views.py

```
@@ -6485,3 +6487,163 @@ def delete_post(request, post_id):
+
+        # Join the room
+        room.participants.add(self.request.user)
+
+        # Get all participants
+        context["participants"] = room.participants.all()
+        context["participants_count"] = room.participants.count()
+
+        return context
+
+
+def delete_voice_chat_room(request, room_id):
```

Ruff (0.8.2)

Missing return type annotation for public function `delete_voice_chat_room`

(ANN201)